audmetric/core/api.py

```diff
@@ -1186,28 +1186,68 @@ def weighted_confusion_error(
 def word_error_rate(
     truth: Sequence[Sequence[str]],
     prediction: Sequence[Sequence[str]],
+    *,
+    norm: str = "truth",
 ) -> float:
     r"""Word error rate based on edit distance.
 
+    The word error rate is computed by aggregating the normalized edit
+    distances of each (truth, prediction)-pair and averaging the
+    aggregated score by the number of pairs.
+
+    The normalized edit distance of each (truth, prediction)-pair is computed
+    as the edit distance divided by a normalization factor n. This represents
+    the average editing cost per sequence item.
+
+    The value of n depends on the norm parameter:
+    If ``norm`` is ``"truth"`` (default):
+    n is set to the reference (truth) length, following the Wikipedia formulation
+    where n is the number of words in the reference (N=S+D+C). This means WER can
```

Comment:
I would either skip (N=S+D+C) or say what S, D, and C stand for.

Reply:
Agree, maybe just change to

```
    n is set to the reference (truth) length.
    This means WER can
```

Reply:
I have followed the suggestion to remove the excess notation.  I have also changed line breaks to (hopefully) make it more readable.

Reply:
Thanks, looks good to me.

If @reichelu agrees, he can resolve here.